### DISCUSSION TITLE
navlist.group: add decoration options like navlist.item

### DISCUSSION THREAD

**Discussion by jbraband** [livewire/flux#239] [Feature Requests]
I think adding the decoration options available for a navlist.item to navlist.group makes a lot of sense. At least, I know I'd use the icon ones and the badge

from ```stubs/resources/views/flux/navlist/item.blade.php```
```
@props([
    'iconVariant' => 'outline',
    'iconTrailing' => null,
    'variant' => null,
    'iconDot' => null,
    'badge' => null,
    'icon' => null,
])
```

**Comment by jbraband** [comment]
I should that I like the icon would go between the chevron and the slot

**Comment by jbraband** [comment]
I just added another Feature Request to add the ability to pass an icon slot to navlist.item (#283). This change would be nice to then also have in navlist.group to have parity between the 2 navlist components

**Comment by kyledoesdev** [comment]
I think it would be nice to be able to have badge-variant to have the normal/solid badge types
